README.md

```diff
@@ -21,36 +17,36 @@ jobs:
     runs-on: self-hosted
     steps:
       - name: Check out repository
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
       - name: Run tests
-        uses: matlab-actions/run-tests@v1
+        uses: matlab-actions/run-tests@v2
 ```
 
-### Generate Test Artifacts on GitHub-Hosted Runner
-Before you run tests and generate artifacts on a GitHub-hosted runner, first use the [Setup MATLAB](https://github.com/matlab-actions/setup-matlab/) action. The action sets up your specified MATLAB release (R2020a or later) on a Linux&reg; virtual machine. If you do not specify a release, the action sets up the latest release of MATLAB.
+### Generate Artifacts on GitHub-Hosted Runner
+Before you run tests and generate artifacts on a [GitHub-hosted runner](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners), first use the [Setup MATLAB](https://github.com/matlab-actions/setup-matlab/) action. The action sets up your specified MATLAB release (R2020a or later) on a Linux&reg;, Windows&reg;, or macOS virtual machine. If you do not specify a release, the action sets up the latest release of MATLAB.
```

Comment:
macOS is a registered trademark. https://www.apple.com/legal/intellectual-property/trademark/appletmlist.html

Reply:
Updated. It's strange that MATLAB documentation has been using macOS without any trademark symbols: https://www.mathworks.com/help/matlab/ref/matlabmacos.html?searchHighlight=macOS&s_tid=srchtitle_support_results_1_macOS. Maybe Apple added the symbol recently.